examples/gameroom/daemon/openapi.yml

```diff
@@ -629,22 +629,31 @@ paths:
           required: true
           schema:
             type: string
+        - name: wait
+          in: query
+          description: Time gameroomd should wait for the batch to be processed. If set to "false" gameroomd will not wait.
+          required: false
+          schema:
+            type: integer
+            default: 30
       requestBody:
         content:
           application/octet-stream:
             schema:
               type: string
               format: binary
       responses:
-          202:
-            description:  The payload was accepted
+          200:
```

Comment:
In other implementations (sawtooth, etc) I think we still have this as a 202

Reply:
I think it should be 200 if the batch status was waited on and it came out as a success, and 202 if wait is set to false. 202 implies that we don't know if the request succeeded or not, just that it was accepted.

Reply:
updated